### PR TITLE
Minor corrections and updates of clarithromycin model, evaluation pla…

### PR DESCRIPTION
**Pull request by AndreDlm**
…n and static content files

### CODE REVIEW

File: Evaluation/Input/evaluation_plan.json [hunk 1]
```diff
@@ -0,0 +1,497 @@
+{
+    "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v2.0/OSP_Qualification_Plan_Schema.json",
```
Comment (Yuri05) [inline Evaluation/Input/evaluation_plan.json]: "$schema": "https://github.com/Open-Systems-Pharmacology/QualificationPlan/releases/download/v2.0/
why not 2.1?

File: Evaluation/Input/evaluation_plan.json [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+    "Projects": [
+        {
+            "Id": "Clarithromycin-Model",
+            "Path": "https://raw.githubusercontent.com/LMFuhr/Clarithromycin-Model/master/Clarithromycin-Model.json",
```
Comment (Yuri05) [inline Evaluation/Input/evaluation_plan.json]: Reference to a project outside of OSP???

**Review by Yuri05 — CHANGES_REQUESTED** [review]
s. comments

**Comment by sfrechen** [comment]
@Yuri05 
Changes implemented

**Pull request merged**